hax/hax/server.py

```diff
@@ -210,6 +212,50 @@ def fn():
     return _process
 
 
+def process_state_update(planner: WorkPlanner, consul_util: ConsulUtil):
+    async def _process(request):
+        data = await request.json()
+
+        loop = asyncio.get_event_loop()
+
+        def fn():
+            proc_state_to_objhealth = {
+                'M0_CONF_HA_PROCESS_STARTING': ObjHealth.OFFLINE,
+                'M0_CONF_HA_PROCESS_STARTED': ObjHealth.OK,
+                'M0_CONF_HA_PROCESS_STOPPING': ObjHealth.OFFLINE,
+                'M0_CONF_HA_PROCESS_STOPPED': ObjHealth.OFFLINE
+            }
+            # import pudb.remote
+            # pudb.remote.set_trace(term_size=(80, 40), port=9998)
```

Comment:
I think we can remove this in the final commit :)

Reply:
Yeah, looks it was left from one of the earlier commit 49317b36, I think it is for debugging purposes. Let's keep it for convenience to be used later.